homework/03LessonThree.js

```diff
@@ -20,15 +20,15 @@ var firstName = 'susan';
 
 // Problem 1:
 // What is the value of firstName at this point in the file?
-var valueOfFirstName;
+var valueOfFirstName = 'susan';
 
 // Problem 2:
 // What is the value of lastName at this point in the file?
-var valueOfLastName;
+var valueOfLastName = 'plaid';
 
 // Problem 3:
 // Fix the following function so that it returns the first argument
-function returnTheFirstArgument(firstArgument) {
+function TheFirstArgument(firstArgument) {
   var firstArgument = 'bob';
```

Comment:
Rey,

This function will always return 'bob' no mater what. If you scroll down and read the tests for this assignment, you'll see that different arguments keep trying to get passed to function returnTheFirstArgument.

If I did ```javascript returnTheFurstARgument('hi')```  It wont return hi, but will return 'bob' because I keep reassigning the value of "firstArgument" to it. To fix this, just comment out the reassignment, "var firstAgument = 'bob';"  If this still doesnt make sense, please come talk to me during class.